dao/src/main/scala/za/co/absa/enceladus/dao/MenasDAO.scala

```diff
@@ -15,62 +15,106 @@
 
 package za.co.absa.enceladus.dao
 
+import org.apache.spark.sql.types.StructType
 import za.co.absa.enceladus.model._
 import za.co.absa.atum.model._
 
+/**
+  * Trait for Menas API DAO.
+  */
+trait MenasDAO {
 
-import scala.util.Try
+  /**
+    * Authenticates user with Menas
+    */
+  def authenticate(): Unit
```

Comment:
Fundamentally these are definitely `checked` exceptions, so it was me just being lazy and breaking convention.

I had made them `RuntimeException`s because Mockito didn't see a checked exception in the method's signature, so it couldn't [`.thenThrow` it.](https://stackoverflow.com/questions/3762047/throw-checked-exceptions-from-mocks-with-mockito)
The `@throws` annotation adds the checked exception to the signature and all is well.